packages/components/src/toolbar/toolbar/index.tsx

```diff
@@ -55,8 +62,10 @@ function UnforwardedToolbar(
 	// `ToolbarGroup` already uses components-toolbar for compatibility reasons.
 	const finalClassName = classnames(
 		'components-accessible-toolbar',
-		className
+		className,
+		variant === undefined ? undefined : `is-${ variant }`
```

Comment:
`classNames()` is flexible and works just fine with both `undefined` and `false` values, so this shorter version will also work well:
```suggestion
		variant && `is-${ variant }`
```

Reply:
Thank you! Much cleaner